.github/ISSUE_TEMPLATE/2-feature-request.yml

```diff
@@ -8,12 +8,11 @@ body:
         Please fill in as much of the template below as you're able.
   - type: textarea
     attributes:
-      label: Is your feature request related to a problem? Please describe.
-      description: Please describe the problem you are trying to solve.
+      label: Is your feature request related to a problem? Please describe if so.
```

Comment:
@ArjunSharda, it seems the previous label and description are good. Let's keep it as it is.

Reply:
ok